test/Bounty.test.js

```diff
@@ -1,113 +1,89 @@
 const { ethGetBalance, ethSendTransaction } = require('./helpers/web3');
+const expectEvent = require('./helpers/expectEvent');
+const { assertRevert } = require('./helpers/assertRevert');
 
-var SecureTargetBounty = artifacts.require('SecureTargetBounty');
-var InsecureTargetBounty = artifacts.require('InsecureTargetBounty');
+const SecureTargetBounty = artifacts.require('SecureTargetBounty');
+const InsecureTargetBounty = artifacts.require('InsecureTargetBounty');
+
+require('chai')
+  .use(require('chai-bignumber')(web3.BigNumber))
+  .should();
 
 const sendReward = async (from, to, value) => ethSendTransaction({
-  from, to, value,
+  from,
+  to,
+  value,
 });
 
-function awaitEvent (event, handler) {
-  return new Promise((resolve, reject) => {
-    function wrappedHandler (...args) {
-      Promise.resolve(handler(...args)).then(resolve).catch(reject);
-    }
+const reward = web3.toWei(1, 'ether');
 
-    event.watch(wrappedHandler);
-  });
-}
+contract('Bounty', function ([_, owner, researcher]) {
+  describe('Against secure contract', function () {
+    beforeEach(async function () {
+      this.bounty = await SecureTargetBounty.new({ from: owner });
+    });
 
-contract('Bounty', function (accounts) {
-  it('sets reward', async function () {
-    let owner = accounts[0];
-    let reward = web3.toWei(1, 'ether');
-    let bounty = await SecureTargetBounty.new();
-    await sendReward(owner, bounty.address, reward);
+    it('sets reward', async function () {
```

Comment:
I'd remove this `it` altogether, it's already being tested for in all other test cases (and if we were to add it, we should have it on both `describe`s).

Reply:
I think it's ok to be explicit here. I've refactored the tests in the next commit to make this cleaner